mat/cholesky.go

```diff
@@ -40,12 +40,13 @@ func (c *Cholesky) updateCond(norm float64) {
 	work := getFloats(3*n, false)
 	defer putFloats(work)
 	if norm < 0 {
-		// This is an approximation. By the definition of a norm, ||AB|| <= ||A|| ||B||.
-		// Here, A = U^T * U.
-		// The condition number is ||A|| || A^-1||, so this will underestimate
-		// the condition number somewhat.
-		// The norm of the original factorized matrix cannot be stored because of
-		// update possibilities.
+		// This is an approximation. By the definition of a norm,
+		//  ‖AB‖ <= ‖A‖ ‖B‖.
```

Comment:
`‖` renders unfortunately poorly in the font chosen by github (though this is not surprising given their other choices in the default style - pastels and low contrast text).